src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -14,10 +18,14 @@ class ForecastDataInterface:
     def __init__(self,
                  forecast_paths: paths.ForecastPaths,
                  regression_paths: paths.RegressionPaths,
-                 covariate_paths: paths.CovariatePaths):
+                 covariate_paths: paths.CovariatePaths,
+                 regression_marshall,
+                 forecast_marshall):
         self.forecast_paths = forecast_paths
         self.regression_paths = regression_paths
         self.covariate_paths = covariate_paths
+        self.regression_marshall = regression_marshall
+        self.forecast_marshall = forecast_marshall
 
     @classmethod
     def from_specification(cls, specification: ForecastSpecification) -> 'ForecastDataInterface':
```

Comment:
the DataInterface classes are all meant to be instantiated `from_specification` so we piggy-back the Marshall objects here to avoid any additional code churn.